src/background/hypothesis-chrome-extension.js

```diff
@@ -107,7 +113,9 @@ export default function HypothesisChromeExtension(dependencies) {
     state.load(store.all());
     chromeTabs.query({}, function (tabs) {
       tabs.forEach(function (tab) {
-        onTabStateChange(tab.id, state.getState(tab.id));
+        if (tab.id) {
+          onTabStateChange(tab.id, state.getState(tab.id));
+        }
```

Comment:
Unlike the previous case I'm not sure that it is safe to assume the tab has an ID, so I added a check. https://developer.chrome.com/extensions/tabs#type-Tab mentions scenarios where it might not.